swap_meet/clothing.py

```diff
@@ -1,2 +1,27 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    '''Clothing Class(inherits from Item Class)
+
+    override __init__
+    attribute: id(unique integer), fabric=“Unknown”
+    super() for id
+
+    __str__()
+    returns "An object of type Clothing with id <id value>. It is made from <fabric value> fabric."
+    '''
+    def __init__(self, condition=0, id=None, fabric="Unknown"):
+        self.fabric = fabric
+        self.condition = condition
```

Comment:
Notice that we have `condition` as an attribute inside of `Item` as well, the parent class. Since `Clothing`, `Electronic`, and `Decor` are all inheriting from `Item` we don't need to make include `condition` here or in the other children classes. Right now, your code as a redundancy by including it. @stephaugi @edbelova 
```suggestion
```